SalesforceCommerceCloud/pwa-kit
# [V2] Hide webpack performance warnings

# Description

Ticket: https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000mQtDRYA0/view

This PR hides the Webpack's performance warnings for the `pwa` package, because we already have a similar `bundlesize` check in `pwa`.

When running `npm run build`:

Before:
![NeoVim Terminal 2022-03-30 at 17 06 02](https://user-images.githubusercontent.com/847300/160950544-4381b5fd-2634-4dc8-a999-c1c24a756014.png)

After: 
![NeoVim Terminal 2022-03-30 at 17 04 16](https://user-images.githubusercontent.com/847300/160950557-93da01f1-f42c-4678-88c7-121f4e4e2e7c.png)

And now running `npm run max-file-size` is also passing:
```bash
npm run test:max-file-size

> scaffold-pwa@2.0.0-dev.2 test:max-file-size /Users/vmarta/repos/pwa-kit/packages/pwa
> bundlesize


─ build/main.js
  ✔ build/main.js      47.73KB < 52 kB gzip

─ build/vendor.js
  ✔ build/vendor.js    240.02KB < 245 kB gzip

  2 checks passed
```

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# How to Test-Drive This PR
1. Switch to 'packages/pwa' directory
3. `npm run build`
5. Verify that the webpack performance hints are no longer showing


# Checklists

## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
